src/warnet/server.py

```diff
@@ -317,10 +317,21 @@ def scenarios_available(self) -> list[tuple]:
             raise ServerError(message=msg) from e
 
     def scenarios_run_file(
-        self, scenario_base64: str, additional_args: list[str], network: str = "warnet"
+        self,
+        scenario_base64: str,
+        scenario_name: str,
+        additional_args: list[str],
+        network: str = "warnet",
     ) -> str:
         scenario_path = None
```

Comment:
nit: I believe we can remove `scenario_path = None`

Reply:
done in d50ba923000b0b84ff895bae92f1469a581bb6ba